website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,14 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
+        // Adapted from http://www.quirksmode.org/js/cookies.html
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+          var cookieName = cookies[i].match(/([^=]+)(?=\=)/);
```

Comment:
The match method will also return null if a match could not be found, so you'll need to write a condition for that.